src/cmd/fetch.go

```diff
@@ -87,6 +133,7 @@ var fetchCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(fetchCmd)
 
-	fetchCmd.Flags().String("branch", "", "Branch name for all repositories")
-	fetchCmd.Flags().String("whitelist", "", "Path to a whitelist file for all repositories")
+	rootCmd.PersistentFlags().String("branch", "", "Branch name for all repositories")
+	rootCmd.PersistentFlags().String("whitelist", "", "Path to a whitelist file for all repositories")
+	rootCmd.PersistentFlags().String("repo-file", "", "Path to a file containing the list of repositories to fetch")
```

Comment:
Why are we again using `PersistentFlags`?

Reply:
I will cherry pick the commit in if you can find the SHA for me

Reply:
I have cherry picked the commit :+1: